flake.nix

```diff
@@ -393,16 +389,16 @@
 
             # TODO: inherit and provide script to run all stuff
             # devnet-container-xcvm
+            # NOTE: The devcontainer is currently broken. Please use the developers devShell
```

Comment:
it is NOT broken on ubuntu x86. I send screenshots 2 times that RA works here. please add ARM into comment

Reply:
Oh sorry yes I meant for ARM, my apologies